core/modules/path/tests/path_pattern.test

```diff
@@ -115,10 +115,10 @@ class PathPatternTestHelper extends BackdropWebTestCase {
     return $term;
   }
 
-  function assertEntityPattern($entity_type, $bundle, $language = LANGUAGE_NONE, $expected) {
+  function assertEntityPattern($entity_type, $bundle, $langcode, $expected) {
```

Comment:
In theory that would change the logic, in practice this function is only used twice in Backdrop and both times the value is provided. Yes, langcode is the better variable name here.